cli/cmd/release_create.go

```diff
@@ -160,6 +165,11 @@ func (r *runners) releaseCreate(cmd *cobra.Command, args []string) (err error) {
 	}()
 
 	log := logger.NewLogger(r.w)
+	if r.outputFormat == "json" {
+		// suppress spinner/log lines for machine-readable output
+		log.Silence()
+		r.w = tabwriter.NewWriter(io.Discard, 0, 0, 0, ' ', 0)
```

Comment:
what happens if it prompts on line 201?

```
confirmed, err = promptForConfirm()
```

Reply:
no output plus invisible spinner
I think I'll remove the zero writer change commit if that's all right? Least bad option barring a major refactor IMO